docs/monitoring/03-01-alertmanager.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Alertmanager
+type: Details
+---
+
+Alertmanager receives and manages alerts coming from Prometheus. It can then forward the notifications on fired alerts to specified channels, such as Slack or VictorOps. 
+
+## Alertmanager configuration
+
+You can configure and manage Alertmanager using the following files:
```

Comment:
Maybe we could add which configuration file is required in which cases?

Reply:
I reworked this one :)